packages/atomic/vitest-utils/testing-helpers/fixtures/headless/commerce/breadcrumb-manager-subcontroller.ts

```diff
@@ -3,6 +3,7 @@ import type {
   BreadcrumbManagerState,
 } from '@coveo/headless/commerce';
 import {vi} from 'vitest';
+import {genericSubscribe} from '../common';
```

Comment:
No big deal, especially since these are just test helpers / fixtures but should we use path aliases for these imports?